### PR TITLE
Even more generalizations

### PR DESCRIPTION
**Pull request by yuvipanda**
This finally makes the chart fully usable from a blank slate!

We need to make some changes to our config repo before deploying this tho.

### CODE REVIEW

File: helm-chart/values.yaml [hunk 1]
```diff
@@ -34,34 +34,28 @@ publicIP: null
 
 # Google OAuth secrets
```
Comment (ryanlovett) [inline helm-chart/values.yaml:35]: Remove comment?

File: helm-chart/values.yaml [hunk 2]
```diff
@@ -34,34 +34,28 @@ publicIP: null
...
 
 token:
   proxy: 'generate with pwgen 64 and put here'
   culler: 'generate with openssl rand -hex 32 here'
 
 singleuser:
   storage:
-    type: dynamic
+    type: none
     capacity: 10Gi
```
Comment (ryanlovett) [inline helm-chart/values.yaml:46]: What do capacity and homeMountPath mean in the context of storage type none?

**Review by ryanlovett — COMMENTED** [review]
Everything lgtm. I had a question about what happens when storage type is none.

**Comment by yuvipanda** [comment]
@ryanlovett I'm going to merge this now, and do a separate commit cleaning up docs. I've deployed this.

**Pull request merged**